robottelo/cli/factory.py

```diff
@@ -1171,14 +1171,17 @@ def make_usergroup(options=None):
 @cacheable
 def make_usergroup_external(options=None):
     """
-    Usage:
+    Usage::
+
         hammer user-group external create [OPTIONS]
 
-    Options:
+    Options::
+
         --auth-source-id AUTH_SOURCE_ID           ID of linked auth source
         --name NAME                               External user group name
         --user-group, --usergroup USER_GROUP_NAME Name to search by
         --user-group-id, --usergroup-id USER_GROUP_ID
+
```

Comment:
I totally agree, Let me try removing it, had to add it for `make docs-clean docs` to PASS, if not I really don't know how do I fix this. Let me re-run it with removed and the exact error it throws for this.